_component_design/selector-strip.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Selector Strip
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: >
+  Selector Strips allow the user to make a selection from a group of options. This pattern is a more visually prominent alternative to check boxes (multi-select) or radio buttons (single-select).
+preview-image: preview-images/selector-strip.svg
+status: stable
+resource: true
+last-modified: 2017-10-29
+---
+
+{% include toc.html %}
+
+## Overview
+
+<div class="hxRow">
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+### When to use
+
+Use a selector strip when you have a small group of options the user must select from. Selector strips can be functionally identical to radio buttons or check boxes (single-select or multi-select), but are typically used instead of those patterns in situations where the selection effects the rest of the page, alters the data on the page, or where the designer would like to give more visual prominence to the group of options.
```

Comment:
Suggestion, change:
but are typically used instead of those patterns in situations
to
`and are used in situations`

Reply:
Reworded the whole paragraph as the writing was causing me heartache. Now reads, "Use a selector strip on forms requiring a user to select from a small group of action. Selector strips function similarly to [radio buttons]({{site.baseurl}}/components/radio-buttons.html) or [check-boxes]({{site.baseurl}}/components/checkboxes.html) depending on the variation used; single-select or multi-select. Selector strips are best used in situations where the selection alters the data options presented on the page, or when seeking to more visual prominence to group of options.